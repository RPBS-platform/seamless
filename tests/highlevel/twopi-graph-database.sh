rm -f twopi-colored.seamless
python3 simple-pi-database.py
python3 twopi-graph-database.py
python3 ~/seamless-scripts/color-graph.py twopi.seamless twopi-colored.seamless
md5sum twopi-colored.seamless
