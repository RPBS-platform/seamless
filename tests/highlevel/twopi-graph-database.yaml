stores: 
    -
      path: "/tmp/seamless-db"      
      readonly: false
      serve_filenames: true
