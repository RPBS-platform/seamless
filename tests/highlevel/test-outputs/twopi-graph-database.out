227e12888875dc351af91a112e35eeca  twopi.seamless
227e12888875dc351af91a112e35eeca  /tmp/twopi-database.seamless
None
<Silk: None >
compute
<Silk: 6.283185307179586 >
18ad9916938d414527c499ebe7e53407  twopi-colored.seamless
